Makefile

```diff
@@ -17,22 +14,18 @@ help:
 	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: run
```

Comment:
```suggestion
.PHONY: run-flask
```

Reply:
Nice catch! I've fixed this and the other `build` / `bootstrap` discrepancy 👍.